deepchem/feat/rdkit_grid_featurizer.py

```diff
@@ -704,6 +769,234 @@ def __init__(self,
         "SO", "P3", "P", "P3+", "F", "Cl", "Br", "I"
     ]
 
+    self.FLAT_FEATURES = {
```

Comment:
s/FLAT_FEATURES/flat_features/

Reply:
I like uppercase for pseudo static variables that shouldn't be mutated at runtime.